src/org/yccheok/jstock/gui/JStock.java

```diff
@@ -1627,6 +1635,14 @@ private void jRadioButtonMenuItem6ActionPerformed(java.awt.event.ActionEvent evt
             }
         }
     }//GEN-LAST:event_jRadioButtonMenuItem6ActionPerformed
+
+    private void jTable1MouseClicked(java.awt.event.MouseEvent evt) {//GEN-FIRST:event_jTable1MouseClicked
+        if (evt.getClickCount() == 2) {
```

Comment:
Both new UI interaction

+ Pressing enter key will invoke single history chart
+ Pressing enter key will invoke multiple history charts
+ Double mouse click will invoke single history chart

Seems natural to me. I will accept them.